requirements.txt

```diff
@@ -50,3 +50,4 @@ git+git://github.com/graphite-project/whisper.git#egg=whisper
 git+git://github.com/graphite-project/ceres.git#egg=ceres
 whitenoise
 scandir
+urllib3==1.19.1
```

Comment:
👍 for not pinning version of urllib3 here. It's master code now, we can start with any version until we hitting some problems with it.